star-war-character-app/src/redux/FilmSlice/FilmSlice.tsx

```diff
@@ -0,0 +1,80 @@
+import { createSlice } from "@reduxjs/toolkit";
+import constant from "../../config/constant";
+import { getFilmActions, getIndividualFilmActions } from "./FilmAsyncThunk";
+import { FilmList } from "./FilmType";
+
+const initialSpecificFilm={
+    title: "",
+    episode_id: constant.defaultUserId,
+    characters: [],
+    created: "",
+    director: "",
+    edited: "",
+    opening_crawl: "",
+    producer: "",
+    release_date: "",
+    url: "",
+    starships: [],
+    species: [],
+    planets: [],
+  }
+
+const initialState: FilmList = {
+list: [],
```

Comment:
Code should be formatted

Reply:
I have resolved this issue in this PR https://github.com/Harsh-Upadhyay-AT/Star-War-Character-App/pull/2 Please check it.